README.md

```diff
@@ -30,15 +32,23 @@ Instead, you also need to make use of one more plugins which will add or extend
 
 By far the most common case will be installing the [eslint-plugin-typescript](https://github.com/nzakas/eslint-plugin-typescript) plugin, but there are also other relevant options available such a [eslint-plugin-tslint](https://github.com/JamesHenry/eslint-plugin-tslint).
 
-## Options
+## Configuration
+
+The following additional configuration options are available by specifying them in [`parserOptions`](https://eslint.org/docs/user-guide/configuring#specifying-parser-options) in your ESLint configuration file.
+
+**`jsx`** - default `false`. Enable parsing JSX when `true`. More details can be found [here](https://www.typescriptlang.org/docs/handbook/jsx.html).
```

Comment:
There are technically other options (`range`, `loc`, `comment`, etc.), but these aren't particularly useful for the end user, since ESLint supplies them. I think we should document the API (`parse()`, `parseForESLint()`, etc.) and document those options there.